### PR TITLE
Boolean 'false' fields are not displayed

### PR DESCRIPTION
**Pull request by emhoracek**
Also, update to use latest Larceny.

### CODE REVIEW

File: spec/Common.hs [hunk 1]
```diff
@@ -86,6 +86,12 @@ department = object [ "id" .= (2 :: Int)
                     , "name" .= ("Sports" :: Text)
                     ]
 
+boolFieldTrue :: Value
+boolFieldTrue = object [ "person" .= object [ "name" .= ("Ada Lovelace" :: Text )]]
+
+boolFieldFalse :: Value
+boolFieldFalse = object [ "person" .= False ]
```
Comment (dhartunian) [inline spec/Common.hs:93]: i think in our case the value is the string `"false"` not the boolean `false`

File: src/Web/Offset/Splices.hs [hunk 2]
```diff
@@ -104,7 +104,8 @@ jsonToFill (String s) = rawTextFill s
 jsonToFill (Number n) = case floatingOrInteger n of
                           Left r -> rawTextFill $ tshow (r :: Double)
                           Right i -> rawTextFill $ tshow (i :: Integer)
-jsonToFill (Bool b) = rawTextFill $ tshow b
+jsonToFill (Bool True) = rawTextFill $ tshow True
+jsonToFill (Bool False) = rawTextFill "<!-- JSON field found, but value is false. -->"
```
Comment (dhartunian) [inline src/Web/Offset/Splices.hs:108]: I dont think we want *all* boolean false values to render this, no?

**Review by dhartunian — CHANGES_REQUESTED** [review]

File: spec/Common.hs (hunk 1 above)
Comment (emhoracek) [reply spec/Common.hs:93]: I don't think so... I did check but let me verify.

File: src/Web/Offset/Splices.hs (hunk 2 above)
Comment (emhoracek) [reply src/Web/Offset/Splices.hs:108]: I can't imagine ever wanting "false" (or even "true")

Comment (emhoracek) [reply src/Web/Offset/Splices.hs:108]: Do you have another idea?

**Review by dhartunian — APPROVED** [review]

**Pull request merged**